presto-ranger-plugin/src/main/resources/ranger-hive-audit.xml

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+    <property>
+        <name>xasecure.audit.is.enabled</name>
```

Comment:
While trying this change, all queries failed with access denied. Turned out that these xml configs part of resources are overriding any configs user provides. E.g. service.name which wasnt preprod_dp_hive in my case was being picked from here causing silent failure in fetching policy and leading to access denied for all operations.

We should remove these configs.

Reply:
acked